scripts/capture.py

```diff
@@ -56,7 +56,9 @@ def matches(frame, threshold, proportion=3):
               help='Specify path and get True/False for bright/dark image')
 @click.option('--rotate', default=0,
               help='Rotate image (0 to 360 degrees)')
-def capture_and_save(warmup, threshold, path, image, rotate):
+@click.option('--print_path', default=False,
```

Comment:
@Adman nice! Maybe the CLI parameter could be called `--print-path` for consistency? Its value will still make it to the `print_path` variable.